cms/templates/maintenance/index.html

```diff
@@ -0,0 +1,17 @@
+<%page expression_filter="h"/>
+<%inherit file="base.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%block name="title">${_('Maintenance Dashboard')}</%block>
+<%block name="viewcontent">
+<section class="container maintenance-content">
```

Comment:
So Should I add some paragraph describing Maintenace Dashboard or should I make this one `div` too ?